bftengine/src/bftengine/KeyManager.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "KVBCInterfaces.h"
+#include "Serializable.h"
+
+class KeyManager {
+ public:
+  struct KeyExchangeMsg : public concord::serialize::SerializableFactory<KeyExchangeMsg> {
+    std::string key_;
```

Comment:
If it is public member, please remove `_`.

Reply:
I don't mind doing it, but I'm not familiar with this convention, I do actually prefer marking all members with _

Reply:
Here is the convention:
https://google.github.io/styleguide/cppguide.html#Variable_Names.
Please change all over.

Reply:
Actually, I'd really like this class to be conformant with the code style.
AFAIK, underscores are not used for public members in the project.

Reply:
I can find examples with `_` for example 
`struct Msg {
  std::unique_ptr<char[]> msg_;
  uint32_t len_;
  uint16_t to_;
};`
In my opinion, it's too pedantic and missing the point of code review to nag on nit stuff